go/oasis-node/cmd/identity/identity.go

```diff
@@ -50,7 +50,7 @@ func doNodeInit(cmd *cobra.Command, args []string) {
 		)
 		os.Exit(1)
 	}
-	if _, err = identity.LoadOrGenerate(dataDir, nodeSignerFactory); err != nil {
+	if _, err = identity.LoadOrGenerate(dataDir, nodeSignerFactory, true); err != nil {
```

Comment:
Does this mean that nodes provisioned this way won't have ephemeral TLS certs unless they obliterate the persisted cert?

Reply:
Yes.  As rotation is off by default for now, I thought this would make most sense.
Once we enable rotation by default, we can also disable persistent TLS certs here or add a flag to control this behavior.